### ISSUE TITLE
Integrate Unused Microsoft Stubs

### ISSUE THREAD

**Issue by ShivnarenSrinivasan** [pandas-dev/pandas-stubs#75]
I am aware that this repo is "born" from the existing stubs, but it seems like there are some stubs that live in the pylance bundle (*I have no idea where on GitHub, though*) that we can probably add in?

For example, the entire `io` folder does not exist in this repo.

Example of `Styler`, for which the `to_html` method does not exist in the pylance bundle..

Path on my machine:
`~\.vscode\extensions\ms-python.vscode-pylance-2022.6.30\dist\bundled\stubs\pandas\io\formats\style.pyi`

![image](https://user-images.githubusercontent.com/71850263/177036982-41b90ec9-0d8c-4fad-8035-d3195096dbb7.png)

Apologies if there is a misunderstanding on my side—also interested in contributing, as I use pandas daily.

**Comment by bashtage** [comment]
I think a lot of these are already in pandas. It might be that these are stripped out for convenience by pylance.

**Comment by ShivnarenSrinivasan** [comment]
> I think a lot of these are already in pandas. It might be that these are stripped out for convenience by pylance.
> 
> I'm pretty sure io is fully typed in pandas.

You're right, they are—but until pandas adds the `py.typed` flag, mypy and pylance are going to have depend on stub files, unless I'm mistaken?

**Comment by Dr-Irv** [comment]
> I am aware that this repo is "born" from the existing stubs, but it seems like there are some stubs that live in the pylance bundle (_I have no idea where on GitHub, though_) that we can probably add in?

A bit of history here.  It used to be that in https://github.com/microsoft/python-type-stubs, the complete distribution of the pandas stubs that were shipped with pylance were included.

At some point in time, only some of them were in that repo that passed the typing tests, but Microsoft would add the "missing" stubs back (but I don't know from where) when packaging `pylance`.

When we took the Microsoft stubs and made this project, we pulled the ones in that repo, but missed the ones Microsoft was merging in.  I'm planning on fixing that today (7/3) or tomorrow.

Related issue is #55
